src/ansys/speos/core/source.py

```diff
@@ -106,6 +110,129 @@ def __init__(
             self.source_template_link = self._project.client[source_instance.source_guid]
             self._reset()
 
+    class Luminous:
```

Comment:
Generate A flux class.
has a property value and a set luminous and set radiant value

what do you think about that?

Reply:
by default it is luminous if you use the set radiant value it take sthe current value and puts ist to radiant and dependdant on the state the value setter works.

i think user experience would be better

Reply:
@StefanThoene @echambla thanks for the discussion, then I will implement the flux class which stays consistent with the other "set_" methods naming. Within the set_flux, the flux type will be changed via set_luminous/set_radiant/set_luminous_intensity, while a value getter and setter property to change the value. Please comfirm it is ok with you both.

Reply:
fine by me